docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: Alluxio
+nav_order: 5
+parent: Getting-Started
+---
+# Get Started with RAPIDS with Alluxio
+
+Alluxio is a data orchestration platform brings your data closer to compute across clusters,
+regions, clouds, and countries. This guide will go through how to set up the RAPIDS Accelerator
+for Apache Spark 3.0 with Alluxio and on premise cluster.
+
+## Prerequisites
+
+Assuming that user has successfully run the RAPIDS Accelerator with on premise cluster according
+to [this doc](getting-started-on-prem.md)
+
+This guide takes Yarn cluster as an example and assumes that there are 2 datanodes and 1 namenode in Yarn cluster, and their hostnames are respectively
+
+``` json
+namenode_hostname
+datanode1_hostname
+datanode2_hostname
+```
+
+## Alluxio setup
+
+It is recommending to deploy Alluxio workers in each datanode and Alluxio master in namenode.
```

Comment:
similar question as above - explain why if this is the case, I assume you actually want it on the YARN nodes so its close to compute, yarn could run on different nodes then hdfs so we would want Nodemanager and Resourcemanager, again explain why

Reply:
yes, you're right